openverse_catalog/dags/common/loader/sql.py

```diff
@@ -129,18 +129,31 @@ def load_local_data_to_intermediate_table(
     _clean_intermediate_table_data(postgres, load_table)
 
 
+def _handle_s3_load_result(cursor) -> int:
```

Comment:
Why would AWS have such a weird return???
The latest Metropolitan Museum run failed because of this.

Reply:
No clue! :sweat_smile: which means it's on-par for AWS lol. That run is what spawned #469